subsys/partition_manager/Kconfig

```diff
@@ -143,14 +143,12 @@ config BUILD_WITH_TFM
 	select PM_SINGLE_IMAGE
 	select ENTROPY_CC3XX
 	select NORDIC_SECURITY_BACKEND
-	select MBEDTLS_X509_LIBRARY
-	select MBEDTLS_TLS_LIBRARY
-	select MBEDTLS_ENABLE_HEAP
+	imply MBEDTLS_ENABLE_HEAP
```

Comment:
I placed comments in the relevant commits where this should be cleaned up, in which case this commit (071e18f6a594ab66b7be6e98606cd1e7e04da4fc) can be dropped.

Reply:
Will fix (cleanup before run)

Reply:
Fixed